### PR TITLE
JBIDE-20709: openshift 3 features

### PR DESCRIPTION
**Pull request by fbricon**
Signed-off-by: Fred Bricon <fbricon@gmail.com>

### CODE REVIEW

File: features/openshift3.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+:page-feature_tagline: The next gen Cloud
+
+== OpenShift 3
+=== Discover the new OpenShift
+image::images/openshift3_create_app.png[]
+
+JBoss Tools presents to you a TechPreview version or the OpenShift 3 tooling for Eclipse.
+Installable from JBoss Central, it will allow you to manage your OpenShit 3 Projects,
+deploy new applications in the Cloud,
+using pre-defined (or your own) templatesm, or even Docker images. You will be able to import
```
Comment (alexeykazakov) [inline features/openshift3.adoc]: templatesm -> templates

File: features/openshift3.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+:page-feature_image_url: images/openshift_icon_256px.png
+:page-feature_highlighted: true
+:page-feature_order: 2
+:page-feature_tagline: The next gen Cloud
+
+== OpenShift 3
+=== Discover the new OpenShift
+image::images/openshift3_create_app.png[]
+
+JBoss Tools presents to you a TechPreview version or the OpenShift 3 tooling for Eclipse.
```
Comment (mishaone) [inline features/openshift3.adoc]: Original: JBoss Tools presents to you a TechPreview version or the OpenShift 3 tooling for Eclipse.
Recommended: JBoss Tools presents a Technical Preview version of the OpenShift 3 Tooling for Eclipse.

**Comment by mishaone** [comment]
The rest looks good. :)

**Comment by alexeykazakov** [comment]
Thanks!. Fixed and pushed to master and production.

**Pull request closed without merging**